plugins/workspace-minimap/src/minimap.ts

```diff
@@ -91,24 +91,25 @@ export class Minimap {
     /**
      * Converts a mouse event on the minimap into scroll coordinates for
      * the primary viewport.
+     * @param primaryMetrics The metrics from the primary workspace.
+     * @param minimapMetrics The metrics from the minimap workspace.
      * @param offsetX The x offset of the mouse event.
      * @param offsetY The y offset of the mouse event.
      * @returns (x, y) primary workspace scroll coordinates.
      */
-    private minimapToPrimaryCoords(
-        offsetX: number, offsetY: number): [number, number] {
-      // Get the metrics from the workspaces
-      const primaryMetrics = this.primaryWorkspace.getMetrics();
-      const minimapMetrics = this.minimapWorkspace.getMetrics();
-
-      // Calculates the location of the click relative to the
+    static minimapToPrimaryCoords(
+        primaryMetrics: Blockly.utils.Metrics,
+        minimapMetrics: Blockly.utils.Metrics,
+        offsetX: number,
+        offsetY: number): [number, number] {
+      // Gets the click location relative to the
       // top left of the minimap content.
       offsetX -= (minimapMetrics.svgWidth - minimapMetrics.contentWidth) / 2;
       offsetY -= (minimapMetrics.svgHeight - minimapMetrics.contentHeight) / 2;
 
-      // Calculates the scale between the minimap and primary workspace
-      // and applies it to the offset.
-      const scale = primaryMetrics.contentWidth / minimapMetrics.contentWidth;
+      // Scales the click location into the primary workspace.
```

Comment:
nit: Might be clearer as `coordinate` instead of `click` since presumably it could be some other type of non-click event.